types.go

```diff
@@ -18,6 +20,10 @@ type ReleaseContent struct {
 
 // ReleaseHistory returns version information about a Helm Release.
 type ReleaseHistory struct {
+	// AppVersion is the app version of the Helm Chart that has been deployed.
```

Comment:
Can we have examples for the versions? Or are all of these only semver? I cannot really make sense of this without understanding the whole stack. For instance, I have issues understanding what the difference between the app version and version of helm chart actually is. People told me already but my brain forgot. :D

Reply:
I also don't understand the difference between that and the `Version`.

Reply:
with our own components we can sync our version of the component and the chart version, however, with 3rd party apps like prometheus for example there's a version of the app like 2.6.0 (for prom) and a version of our chart that changes separately from the appVersion, e.g. if we change some flag or other stuff around the app. current example would be that our NGINX-IC chart version is at 0.3.2 and the appVersion is at 0.21.0.

We might be able to phrase that in a clearer way in the comments and maybe also mention that for our own components this is the same.

btw, @rossf7 as our charts often include more than one component this appVersion can only be the main component like prom here (grafana and exporters and stuff are not shown here). I think that's ok, but maybe that's also sth to clarify, that appVersion is the version of the main app in the app package

Reply:
See https://github.com/giantswarm/apiextensions/pull/190#discussion_r249089157 I also updated the godoc here. WDYT?

Reply:
I think this property sucks because it allows to specify only a single component :D 
And it only makes sense when there are multiple...

But that's just a rant.

Reply:
Yes it's maybe a bit messy but it can be useful and its what's available via the Helm Chart format.